app/views/world_location_news/index.html.erb

```diff
@@ -21,78 +21,107 @@
         </div>
       </div>
     </div>
-  </div>
 
-  <div class="block news-block">
-    <div class="inner-block">
-      <section class="featured-news items-<%= @feature_list.current_feature_count %>" id="featured-documents">
-        <% if @feature_list.any_current_features? %>
-          <%= render partial: 'shared/featured',
-            collection: @feature_list.current_featured,
-            as: :feature
-          %>
-        <% end %>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Featured",
+          font_size: "l",
+          margin_bottom: 4,
+          padding: true,
+          border_top: 2,
+        } %>
+        <section class="featured-news items-<%= @feature_list.current_feature_count %>" id="featured-documents">
+          <% if @feature_list.any_current_features? %>
+          <div class="govuk-grid-row govuk-!-padding-bottom-8">
+            <%= render partial: 'shared/featured',
+              collection: @feature_list.current_featured,
+              as: :feature
+            %>
+          </div>
+          <% end %>
+        </section>
+      </div>
+    </div>
 
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds govuk-!-padding-bottom-9">
         <%= render partial: 'shared/recently_updated',
-                  locals: { recently_updated: @recently_updated,
-                            atom_url: atom_feed_url_for(@world_location),
-                            extra_class: 'panel',
-                            see_all_link: latest_path(world_locations: [@world_location]),
-                            documents_source: Locale.current.english? ? 'rummager' : nil } %>
-      </section>
+          locals: {
+            recently_updated: @recently_updated,
+            atom_url: atom_feed_url_for(@world_location),
+            see_all_link: latest_path(world_locations: [@world_location]),
+            documents_source: Locale.current.english? ? 'rummager' : nil 
+        } %>
+      </div>
     </div>
-  </div>
 
-  <div class="block our-mission">
-    <div class="inner-block">
-      <section class="article-section">
-        <h1 class="keyline-header"><%= t('world_location.headings.mission') %></h1>
-        <p>
-          <span class="mission_statement"><%= govspeak_to_html(@world_location.mission_statement) %></span>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds world-locations__our-mission govuk-!-margin-bottom-3">
```

Comment:
You might've already explained this to me but why do we have override classes littered throughout `grid-row`'s on this page? My broad impression is that it's to adhere to the design/the current layout of the page? If that's the case I'm pushing back, we have a consistent spacing matrix for the entire site and I think it's unwise to bypass that for the sake of this one page.

Reply:
>My broad impression is that it's to adhere to the design/the current layout of the page?   

So! It’s the opposite - it’s to more closely adhere to the current re-design:
There’s a lot more breathing space on this between the two sections.  On re-review I’m more tempted to add _more_ space to this.  

Redesign (Left) > Update (Right)
![Screenshot-2021-07-23-at-20 47 54](https://user-images.githubusercontent.com/71266765/126965827-ba00dc36-3351-43d8-b180-a0942eda9597.jpg)

> we have a consistent spacing matrix for the entire site 

I’m not sure if this responsive scale / grid system has been overlaid on these designs in perhaps the way that’s expected?
 In this context I felt it aligned with the design system ideology:

>Occasionally, you might need to make minor adjustments like adding or removing spacing to elements of your design. You can use the responsive spacing override classes for this.\

Unless there's another technique that's more apt for adding spacing for a Design change?

Reply:
Aha, that makes sense! I have a feeling that whitehall's own bothersome styling matrix probably messes with this a little bit. I can't think of an alternative right now so I think your approach makes sense.

One last little press though is that when I test this on int, the spacing for the documents section looks a little whacky:

![Screenshot 2021-07-28 at 10 27 38](https://user-images.githubusercontent.com/64783893/127302624-ea81f7c5-d5ac-4359-9ca0-748117a0730e.png)

Reply:
Ace! that's been kept and tweaked.

> One last little press though is that when I test this on int, the spacing for the documents section looks a little whacky:

Missing local data pretty much means "Dev via integration" is the way to go - updated 

![image](https://user-images.githubusercontent.com/71266765/127314583-b025425c-53e7-4261-97f8-2e0ebcfba8ab.png)

Reply:
Ok, one final nitpick and then I promise that's it! The space between the headings in that documents section are pretty colossal. You've got this outrageous gap between "Documents" and "Our announcements" then under all the other headings there's that same big gap. Could we give the subheadings more uniform spacing, like that under the first one (Our announcements) and then squeeze the spacing under the main section heading? Ideally in line with other headings on the page eg: Our mission.

Reply:
Hey ya - I'm happy to update these but I've a feeling these are Design changes rather than Dev issues - I've sent a link to the [final Design](https://www.figma.com/file/6oVdr466OnwiHjdBmqvZAX/Untitled?node-id=0%3A1). Do these exist within this?

Reply:
They do sort of but the design feels more consistent space-wise. At the very least could we align the spacing on all the subheadings in that section please?

Reply:
Hey @owenatgov I've updated to try find a nice halfway to address these comments and maintain the Design.  Let me know what you think (sharing with Design as well)